user_guide_src/source/database/query_builder.rst

```diff
@@ -88,7 +88,7 @@ $query, which can be used to show the results:
 
 .. literalinclude:: query_builder/004.php
 
-Please visit the :doc:`getResult*() method <results>` page for a full
+Please visit the :doc:`getResult() method <results>` page for a full
```

Comment:
I agree to remove `*`. The `*` implies `getResultArray()` and `getCustomResultObject()`, but they are referred in the [getResult()](https://codeigniter4.github.io/CodeIgniter4/database/results.html#getresult) section.

But it is better that the destination is [getResult()](https://codeigniter4.github.io/CodeIgniter4/database/results.html#getresult), not the page.
Can you change the destination?
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/documentation.rst#to-a-section